lib/extras.php

```diff
@@ -189,6 +189,14 @@ function novablocks_get_header_attributes() {
 			'type'    => 'string',
 			'default' => 'logo-left',
 		),
+		'stickyRow' => array(
+			'type'    => 'string',
+			'default' => 'primary'
+		),
+		'shouldBeSticky' => array(
```

Comment:
This attribute seems redundant. The should be a sticky header if any of the header rows are sticky.